independent-publisher-connectors/Safaricom Mpesa Daraja/apiDefinition.swagger.json

```diff
@@ -0,0 +1,620 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "daraja",
+    "description": "A connector that exposes daraja mpesa endpoints",
+    "version": "1.0",
+    "contact": 
+    {
+      "name": "John Muchiri, Bernard Karaba",
+      "url": "https://johnmuchirim.github.io/",
+      "email": "jeanfrancaiseharicot@gmail.com",
+      "email": "bkaraba14@gmail.com"
```

Comment:
@johnmuchiriM please have both emails in one property. please see below.
 "email": "jeanfrancaiseharicot@gmail.com, bkaraba14@gmail.com"